src/seedu/addressbook/data/person/Printable.java

```diff
@@ -0,0 +1,5 @@
+package seedu.addressbook.data.person;
+
+public interface Printable {
+	String getPrintableString();
```

Comment:
Interface should have header comments. 
Indentation problem here?. Our coding standard requires you to use 4 spaces instead of tabs. You can configure Eclipse to convert tabs to spaces automatically.